src/Traits/Resettable.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CodeIgniter\Shield\Traits;
+
+use CodeIgniter\Shield\Authentication\Authenticators\Session;
+use CodeIgniter\Shield\Models\UserIdentityModel;
+
+/**
+ * Reusable methods to help the
+ * enforcing of password resets
+ */
+trait Resettable
+{
+    /**
+     * Returns true|false based on the value of the
+     * force reset column of the user's identity.
+     */
+    public function requiresPasswordReset(): bool
+    {
+        $identity_model = model(UserIdentityModel::class);
+        $identity       = $identity_model->getIdentityByType($this, Session::ID_TYPE_EMAIL_PASSWORD);
+
+        return $identity->force_reset;
+    }
+
+    /**
+     * Force password reset
+     */
+    public function forcePasswordReset(): bool
```

Comment:
```suggestion
    public function forcePasswordReset(): void
```

Reply:
I applied your suggestion but it returned a phpstan error:

```
PHP Fatal error:  A void function must not return a value in 
/home/runner/work/shield/shield/src/Traits/Resettable.php on line 44 
```

Reply:
Of course, the implementation must also be modified to throw an exception on error 
and return nothing on success.